bin/run-in-docker.sh

```diff
@@ -42,5 +42,6 @@ docker run \
     --read-only \
     --mount type=bind,src="${solution_dir}",dst=/solution \
     --mount type=bind,src="${output_dir}",dst=/output \
+    --mount type=bind,src="${PWD}/bin",dst=/opt/test-runner/bin \
```

Comment:
This won't work inn production, as you can't change the command that runs the Docker image. The solution to this is simple: just copy the contents of the bin directory into the docker image from within the dockerfile (analogous to https://github.com/exercism/csharp-test-runner/blob/main/Dockerfile#L48).

Reply:
I did wonder but then had a look in the python test runner and theirs was different. Docker was complaining that it couldn't find /opt/test-runner/bin/dyalog-apl-runner.apls (despite the whole repo being copied as part of the Dockerfile). In fact, it is generally confusing to me that we copy all of the repo but then mount the bin directory? I understand mounting the output and solution as they presumably come from somewhere else in the actual system.

Anyway, I'll reset this script, have another go and let you know when it's working again.

Reply:
> In fact, it is generally confusing to me that we copy all of the repo but then mount the bin directory

But we don't mount the bin directory, this PR does :)

Reply:
I've reset the docker scripts. Chalk it up to general confusion about Docker and permissions (especially permissions on my machine vs. permissions within the container.